SparseMap3D[] maps
geometry_msgs/Pose[] origins
